kratos/python_scripts/new_linear_solver_factory.py

```diff
@@ -63,12 +63,7 @@ def ConstructSolver(configuration):
     elif(solver_type == "complex_pastix_solver"):
         import KratosMultiphysics.ExternalSolversApplication
         linear_solver = KratosMultiphysics.ExternalSolversApplication.PastixComplexSolver(configuration)
- 
-    ################################## following solvers need importing the MKLSolversApplication
-    elif (solver_type == "ParallelMKLPardisoSolver"):
```

Comment:
let's emulate the interface through eigen. can we?

Reply:
Should be possible

Reply:
done